src/moonlink_backend/src/lib.rs

```diff
@@ -159,14 +160,15 @@ where
     }
 
     /// Get the current mooncake table schema.
+    /// If the requested database or table doesn't exist, return [`TableNotFound`] error.
```

Comment:
Currently the `TableNotFound` error is an internal connector error, I will work on better propagation.